casbin/model/policy.py

```diff
@@ -90,14 +90,33 @@ def update_policy(self, sec, ptype, old_rule, new_rule):
 
         return self.remove_policy(sec, ptype, old_rule) and self.add_policy(sec, ptype, new_rule)
 
-    def update_policies(self, sec, ptype, old_rules, new_rules):
-        """update policy rules from the model."""
+    def update_policy(self, sec, ptype, old_rule, new_rule):
+        """update a policy rule from the model."""
 
-        for rule in old_rules:
-            if not self.has_policy(sec, ptype, rule):
-                return False
+        if sec not in self.model.keys():
+            return False
+        if ptype not in self.model[sec]:
+            return False
+
+        ast = self.model[sec][ptype]
+
+        # find rule index, if fail return false
+        try:
+            rule_index = ast.policy.index(old_rule)
+        except:
+            return False
+        # find rule priority index, if fail, add directly.
+        try:
+            priority_index = ast.tokens.index(old_rule)
+        except ValueError:
+            ast.policy[rule_index] = new_rule
+        else:
+            if old_rule[priority_index] == new_rule[priority_index]:
+                ast.policy[rule_index] = new_rule
+            else:
+                raise Exception("New rule should have the same priority with old rule.")
```

Comment:
You overwrite `update_policies`, but not `update_policy`

Reply:
I noticed, but I am not familiar with git, I try to repush and fail. @.@

Reply:
I am sorry about wasting of your time.

Reply:
I want to change the remote branch without a new commit, so I first rename the old brach, and delete the remote brach, then this pr is closed. I will put a new pr latter after my careful check. Sorry.

Reply:
You can do interactive rebase then force push to your origin to keep only one commit.